app/services/CreditHistoryService.scala

```diff
@@ -63,18 +65,24 @@ class CreditHistoryService @Inject()(incomeTaxViewChangeConnector: IncomeTaxView
     }
   }
 
+  def getTaxYearAsInt(taxYear: String): Int =
```

Comment:
method to be remove during https://jira.tools.tax.service.gov.uk/browse/MISUV-4508 implementation